DataConnectors/digital shadows/TimerTrigger1/DS_api.py

```diff
@@ -0,0 +1,35 @@
+""" handles all DS api related functions """
+import requests
+import base64
+
+
+class api:
+    
+    url = "https://api.searchlight.app/v1/"                 #main url for api
+
+    """ constructer initializes the DS creds """
+    def __init__(self, id, key, secret):
+        passkey = key + ":" + secret
+        self.id = id
+        self.b64val = base64.b64encode(bytes(passkey, 'utf-8')).decode("ascii")
+
+    def get_alerts(self, alert_id):
+        alert_url = self.url + "alerts?id=" + str(alert_id)
+        response = requests.get(alert_url, headers={"Authorization": "Basic %s" % self.b64val, "searchlight-account-id": "%s" % self.id})
```

Comment:
Please make sure all requests call have a timeout. Actually it will be good idea to write a request wrapper function, so that all calls have the timeout, header.